notes/v1.0.0-M8.md

```diff
@@ -27,7 +27,29 @@ In some cases, bloop could get hanging compilation processes caused by `CTRL-C`
 
 [Such a bug](https://github.com/scalacenter/bloop/issues/379) is now fixed. We have experimented with file watching in several projects and confirmed it now works reliably.
 
-#### [Nailgun exception is gone][#390]
+#### [Start the server with `bloop server`][#400]
+
+You can now start the server from the bloop client with `bloop server`. This improvement was suggested by both Olafur and Jon and has the benefits of not polluting the space for autocompletions and yet having an easy way to start it, without need to remember the weird `blp-server` binary name.
+
+#### [Test filters are now accessible via `--only`][#399]
+
+The old `bloop test foo -f com.my.TestSuite` has been replaced by `bloop test foo -o com.my.TestSuite` (`-o` is named after `--only`). This change brings more ergonomics and makes it intuitive for those users that are used to the `testOnly` style in sbt.
+
+If you want to specify more than one test filter, type several `-o` arguments. Example: `bloop test foo -o com.myTestSuite -o ch.epfl.scala.CoolSuite`.
+
+#### [Improvements to the sbt plugin][#397]
+
+The sbt bloop plugin has several improvements over the plugin in M7:
+
+- [Support for source dependencies][#394]: if you use source dependencies, set `bloopAggregateSourceDependencies in Global := true` and run `bloopInstall`. `sbt-bloop` will then write all the configuration files to the same bloop config directory, and not in the base directories of the build of every sourcedep.
+- [`sbt-bloop` is now less verbose][#370] when generating configuration files.
+- [Workaround for missing `bloopInstall` autocompletion][#373]. Bloop works around a bug in sbt that doesn't autocompletes settings scoped globally.
+
+#### Extract classpath options from sbt
+
+If you have configured classpath options in your sbt build, bloop will now extract them from your build. See [#110][].
```

Comment:
This hasn't been merged yet

Reply:
Yeah, I noticed. This will be merged when the queue is free. I actually have to remove it since you already added notes for it in your PR.